src/java/za/co/mip/ablduck/ABLDuck.java

```diff
@@ -169,7 +169,6 @@ public void setDataFilesOnly(Boolean dataOnly) {
 
     @Override
     public void execute() {
-        checkDlcHome();

```

Comment:
Why did you remove that ? Every PCT task should call this method around the beginning of execute()

Reply:
We don't need DLC path to execute this task, so I just thougth we could remove it because in this case it's useless.
_(and it alowed me to test this task with a computer without openedge on it)_

But I can put it back if you prefer :)

Reply:
You're right.